README.adoc

```diff
@@ -328,9 +328,9 @@ You will receive a response that contains an array of all the members in your cr
 
 === Implementing the update operation
 
-The [hotspot=update]`update()` method handles the implementation of the `update` operation. After `Crew` collection is retrieved, a document is created with the specified object id to query the collection. Next, a new crew member [hotspot=crewMemberUpdate]`Document` is created with the updated configuration. The [hotspot=replaceOne]`MongoCollection.replaceOne()` method is called with the query and new crew member document, which updates all matching queries with the new document. Because the object id is unique in the collection, only one document is updated. `The MongoCollection.replaceOne()` method also returns an UpdateResult, which determines how many documents matched the query. If the match count is zero, then the object id doesn't exist.
+The [hotspot=update]`update()` method handles the implementation of the update operation. After `Crew` collection is retrieved, a document is created with the specified object id to query the collection. Next, a new crew member [hotspot=crewMemberUpdate]`Document` is created with the updated configuration. The [hotspot=replaceOne]`MongoCollection.replaceOne()` method is called with the query and new crew member document, which updates all matching queries with the new document. Because the object id is unique in the collection, only one document is updated. `The MongoCollection.replaceOne()` method also returns an UpdateResult, which determines how many documents matched the query. If the match count is zero, then the object id doesn't exist.
```

Comment:
Break into multiple lines. 

Also not sure if we should put `id` in backticks.